sql/src/main/java/io/crate/operation/projectors/UpdateProjector.java

```diff
@@ -65,16 +63,15 @@
     private final ESLogger logger = Loggers.getLogger(getClass());
 
     public UpdateProjector(ShardId shardId,
-                           TransportUpdateAction transportUpdateAction,
-                           List<ColumnIdent> columnIdents,
-                           CollectExpression<?>[] collectExpressions,
+                           TransportShardUpdateAction transportUpdateAction,
+                           CollectExpression<?> collectUidExpression,
+                           Map<String, Symbol> assignments,
```

Comment:
maybe add a javadoc string for the updateProjector explaining that the key in assignments is supposed to be a fqn columnIdent